WorkflowLayer/Fastq2ProteinsWF.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Bio.VCF;
+using Proteogenomics;
+using Proteomics;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using ToolWrapperLayer;
+using UsefulProteomicsDatabases;
+using System;
+
+namespace WorkflowLayer
+{
+    public class Fastq2ProteinsWF : SpritzWorkflow
```

Comment:
I'm a little confused why these `SpritzWorkflow`s are in the `WorkflowLayer`. Aren't they specific to the GUI code, e.g. for dynamically setting the parameters? Maybe they should be in a folder in the GUI project.

Reply:
SpritzFlow is parent class for all other workflows.